index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt a Dog</title>
+    <script src="js/dogs.js" charset="utf-8"></script>
+    <link href="styles/style.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <div class="header-left">
+            <img class="logo" src="images/adoptadog-logo.svg" alt="adoptadog logo">
+            <h1>Seattle’s premiere dog adoption agency</h1>
+        </div>
+        <div class="header-right">
+            <div class="pay">
+                <img class="cart" src="images/cart-icon.svg" alt="shopping cart">
+                <p>$1234.56</p>
+            </div>
+            <button>Checkout</button>
+        </div>
+    </header>
+
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li>/</li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li>/</li>
+            <li><a href="blog.html">Blog</a></li>
+
+        </ul>
+    </nav>
+
+    <section class="adopt">
+
+        <img class="col-1 row-1" src="images/home-intro.jpg" alt="Man hugging a dog">
+
+        <div class="col-2 row-1">
+            <h2>Adopt A Dog</h2>
```

Comment:
I would use flexbox for this over css grid. When changing screen sizes, your image gets tiny and text pressed against the side. Using flexbox, you can leverage `flex-wrap: wrap` to make your text go under your image on a smaller screen size, more details here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/